source/api/presentation/3.0/index.md

```diff
@@ -1179,17 +1179,17 @@ Additional features of the [Web Annotation][webanno] data model _MAY_ also be us
 
 ###  5.6. Range
 
-Ranges are used to describe additional structure within an object, such as newspaper articles that span pages or chapters within a book. Ranges can include Canvases, parts of Canvases, or other Ranges, creating a nested tree structure such as a table of contents.
+Ranges are used to describe additional structure within an object, such as newspaper sections or articles, or chapters within a book. Ranges can include Canvases, parts of Canvases, or other Ranges, creating a tree structure like a table of contents.
 
 The intent of adding a Range to the Manifest is to allow the client to display a hierarchical navigation interface to enable the user to quickly move through the object's content. Clients _SHOULD_ present only Ranges with the `label` property and without the "no-nav" `renderingHint` to the user. Clients _SHOULD NOT_ render Canvas labels as part of the navigation, and a Range that wraps the Canvas _MUST_ be created if this is the desired presentation.
 
-Ranges _MUST_ have URIs and they _SHOULD_ be http(s) URIs.  Ranges are embedded or referenced within the Manifest in a `structures` property. Ranges then embed or reference other Ranges, Canvases or parts of Canvases in the `items` field.  Each entry in the `items` field _MUST_ be a JSON object, and it _MUST_ have the `id` and `type` properties.
+Ranges _MUST_ have URIs and they _SHOULD_ be http(s) URIs.  Ranges are embedded or referenced within the Manifest in a `structures` property. Ranges embed or reference other Ranges, Canvases, or parts of Canvases in the `items` property.  Each entry in the `items` property _MUST_ be a JSON object, and it _MUST_ have the `id` and `type` properties.
 
-All of the Canvases or parts that should be considered as being part of a Range _MUST_ be included within the Range's members, or a descendant Range's members.
+All of the Canvases or parts that should be considered as being part of a Range _MUST_ be included within the Range's `items` list, or a descendant Range's `items`. 
 
-The Canvases and parts of Canvases may or may not be contiguous or in the same order as any Sequence.  Examples include newspaper articles that are continued in different sections, or simply a chapter that starts half way through a page. Parts of Canvases _MUST_ be rectangular and are described using the `xywh=` fragment approach.
+The Canvases and parts of Canvases may or may not be contiguous or in the same order as in any Sequence.  Examples include newspaper articles that are continued in different sections, or simply a chapter that starts half way through a page. Parts of Canvases _MUST_ be rectangular and are described by appending an `xywh=` fragment to the end of the Canvas's URI.
```

Comment:
> "... may or may not be ..."

"need not be" or "are not necessarily" or start the sentence with "It is not necessary for..."

Reply:
Examples need to work for time-only canvas:

Examples include newspaper articles that are continued in different sections, a chapter that starts half way through a page, or time segments of a single canvas that represent different sections of a piece of music. Spatial parts of Canvases _MUST_ be rectangular and are described by appending an `xywh=` fragment to the end of the Canvas's URI. Temporal parts of Canvases are described by appending a `t=` fragment to the Canvas's URI. Temporal and spatial fragments can be combined to identify a particular region for a particular time duration.

(with an example)